src/content_cao.h

```diff
@@ -108,14 +108,22 @@ class GenericCAO : public ClientActiveObject
 	float m_step_distance_counter;
 	u8 m_last_light;
 	bool m_is_visible;
+	f32 m_knockback_timer;
+	bool m_knockback;
+	v3f m_knockback_direction;
 
 	std::vector<u16> m_children;
 
+	void accelerateHorizontal(const v3f &target_speed, const f32 max_increase);
+	void accelerateVertical(const v3f &target_speed, const f32 max_increase);
```

Comment:
These functions also exist in localplayer and it is marked const. But okay I remove.